errno/errname.go

```diff
@@ -1138,6 +1138,7 @@ var MySQLErrName = map[uint16]*mysql.ErrMessage{
 	ErrResourceGroupSupportDisabled:   mysql.Message("Resource control feature is disabled. Run `SET GLOBAL tidb_enable_resource_control='on'` to enable the feature", nil),
 	ErrResourceGroupConfigUnavailable: mysql.Message("Resource group configuration is unavailable", nil),
 	ErrResourceGroupThrottled:         mysql.Message("Exceeded resource group quota limitation", nil),
+	ErrResourceGroupQueryRunaway:      mysql.Message("Query execution was interrupted, identified as runaway query", nil),
```

Comment:
Mabye "Query execution was terminated, identified as a runaway query." is better

Reply:
other places use `interrupted`